_posts/2025-09-10_community-survey-findings.md

```diff
@@ -0,0 +1,178 @@
+# Overview
+In October 2024 the Vega Strike Development team put out a [survey](https://www.vega-strike.org/blog/2024/10/30/community-survey/); and
+today we'd like to share the results for the 53 responses we received. While not a lot of responses, it will be used to help inform
+the team. We'll repeat similar surveys in the future.
+
+The primary purpose of this survey was to help us target our CI system for what builds we focus on and what systems we make releases for.
+Please keep in mind that the questions allowed people to make more than one selection; so the totals between different sections may add up to more than the answers received
+and not match other sections.
+
+## Platform: Operating Systems
+
+We have been building on a lot of platforms as we were (a) guessing where our community might be, (b) trying to target popular platforms, and (c) covering platforms the
+current development team uses themselves. This survey provides us with the first bit of data to refine those based on your feedback.
+
+Results:
+```mermaid
+xychart-beta horizontal
+  title "Platform Usage"
+  x-axis "Platforms" ["Windows 11", "Windows 10", "Windows 8", "Windows 7", "Windows XP", "Mac OSX 15", "Mac OSX 14", "Mac OSX 13", "Mac OSX 12", "Red Hat Linux 9", "Red Hat Linux 8", "OpenSuSE Leap", "Linux Mint 21", "Linux Mint 22", "Ubuntu 20.04", "Ubuntu 22.04", "Ubuntu 24.04", "Fedora Linux", "Arch Linux", "Rocky Linux 9", "Rocky Linux 8", "Debian Bookworm", "Debian Bullseye", "Linux - Snap Packages", "Linux - FlatPak", "Gentoo Linux", "PC Linux OS", "Debian Trixie-Forkie", "Mageia LInux", "Slackware 15", "Slackware"]
+  y-axis "Users" 0 --> 26
+  bar [25,17,2,2,1,1,0,0,0,0,0,0,4,5,3,5,5,3,9,0,0,8,4,0,5,1,1,1,1,1,1]
+  line [25,17,2,2,1,1,0,0,0,0,0,0,4,5,3,5,5,3,9,0,0,8,4,0,5,1,1,1,1,1,1]
+
+```
+
+From this we can see that the majority is currently running on Windows (47 users across the various versions), followed by Linux users across a few different distros (57 users
+across Mint(9), Ubuntu(11), Fedora (3), Debian(13), FlatPak(5), Mageia(1), Slackware(2), and Arch(9)).
+
+Now, at the time of the survey we had only released binaries for the Linux Platforms, and we do not have direct support for FlatPak yet (though we are aware that someone in the
+community has at least tried to make a FlatPak build). So it would be natural to expect to see more Linux users at this time. The primary reason for this support was simply
+that we were still working out kinks in the Windows and Mac builds and had not yet developed installers for them as of Vega Strike 0.8.0. Since then we have released
+Vega Strike 0.9.0 with binaries for Windows and Mac; though our Mac builds are presently limited to the Intel platform.
+
+## Platform: Hardware
+
+We also asked about Hardware Architectures:
+
+```mermaid
+xychart-beta horizontal
+    title "Hardware Architecture Usage"
+    x-axis "Processor Architecture" ["Mac M1/M2/M3 ARM", "Intel x86-64", "ARM (RPi, Snapdragon, etc)", "AMD x64"]
```

Comment:
I think the survey question is wrong. From the [wiki](https://en.wikipedia.org/wiki/X86-64)

> x86-64 (also known as x64, x86_64, AMD64, and Intel 64) is a 64-bit extension of the x86 instruction set. It was announced in 1999 and first available in the AMD Opteron family in 2003.

Therefore, AMD x64 is almost certainly the same as AMD x86-64. I did find a mention of an AMD x64 in reddit, however, it is not an actual CPU:

> X64: Here you have to be a bit careful...sometimes used as short for AMD64/X86-64. But sometimes also used for a special mode of AMD64 processors that run programs in 32 bit address space but with access to all CPU instructions only available for 64 bit applications. (But the later is dying out so most likely it's used as synonym for AMD64/X86-64)

Reply:
<img width="768" height="361" alt="image" src="https://github.com/user-attachments/assets/465ff6c7-c971-4ab2-88c0-a8b9a60d7f31" />

Reply:
Added a note about AMDx64.